### ISSUE TITLE
[Screen Plugin] Enhance with $q.screen.current / $q.screen.currentBreakpoint

### ISSUE THREAD

**Issue by matthiasmeier** [quasarframework/quasar#5693]
It would be great if the $q.screen Plugin could be enhanced with `$q.screen.current` or `$q.screen.currentBreakpoint` which would return the currently active breakpoint as a string.

**Describe alternatives you've considered**
We do currently use the following workaround to get the name/string of the current breakpoint:

```javascript
currentBreakpoint () {
    const breakpoints = ['sm', 'md', 'lg', 'xl']
    for (const breakpoint of breakpoints) {
        if (this.$q.screen[breakpoint]) return breakpoint
    }
    return 'xs'
}
```

**Additional context**
Also, if this would be implemented, it would be a great addition if an according `class` would be added to the `<body>` (e.g. `screen--sm` or the like). This would make styling with additional CSS for specific screen sizes easier.

As often times, being able to just write a nested selector like `.screen--sm .something` is …
_a)_ more convenient than using a `@media` query only to overwrite some styles for specific screen sizes and …
_b)_ the additional class adds specificity to the selector, which makes it easier to overwrite existing rules. (Assuming that overwriting existing rules is the most common use case when writing such styles. Wrapping the same selector in a `@media` query on the other hand does _not_ add specificity to the selector: https://css-tricks.com/how-much-specificity-do-rules-have-like-keyframes-and-media/)

A real life example where we could currently use this is the following: We added a `q-btn` to toggle the drawerRight of our layout and positioned it _inside the drawer_ – some pixels from the top and with a negative left position so that is moved together with the drawer when it slides out. When Quasar switches to the "mobile layout" (`$q.screen.lt.md`), the button then is mis-aligned (since the`drawer` changes to `mobile` and covers the layout header). Instead of adding a `:class` or a specific `:style` to the button in the markup, we could just use 

```scss
.drawer-right-toggle {
    // default styles

    .screen--xs &,
    .screen--sm & {
        // specific styles for small screen sizes
    }
}
```

![Screenshot of the toggle button](https://snipboard.io/eKMJby.jpg)

**Comment by lucasfernog** [comment]
```js
currentBreakpoint () {
	const breakpoints = ['sm', 'md', 'lg', 'xl']
	for (const breakpoint of breakpoints) {
		if (this.$q.screen[breakpoint]) return breakpoint
	}
	return 'xs'
}
```

**Comment by matthiasmeier** [comment]
@lucasfernog Thanks for the idea. Even though this would slightly improve our workaround, I still think that the requested feature would be a good addition to the Screen Plugin.

**Comment by rstoenescu** [comment]
Hi,

Thanks for the suggestion!

Will be available in "quasar" v1.5.2:
* Screen.name, this.$q.screen.name
* New body classes (screen--xs, screen--sm, ...)

Went for the third new sexy variant (commit above) :)
